app/components/Views/confirmations/components/Confirm/AccountNetworkInfo/AccountNetworkInfoCollapsed/AccountNetworkInfoCollapsed.styles.ts

```diff
@@ -15,11 +15,26 @@ const styleSheet = (params: { theme: Theme }) => {
       marginRight: 16,
       alignSelf: 'center',
     },
+    accountInfo: {
+      display: 'flex',
+      flexDirection: 'row',
+      alignItems: 'center',
+    },
     accountName: {
       color: theme.colors.text.default,
       ...fontStyles.bold,
       fontSize: 14,
     },
+    accountLabelWrapper: {
+      backgroundColor: theme.colors.background.alternative,
+      borderRadius: 16,
+      paddingHorizontal: 12,
+      paddingVertical: 2,
+      marginLeft: 8,
+    },
+    accountLabel: {
+      color: theme.colors.text.alternative,
+    },
```

Comment:
```suggestion
    accountLabelWrapper: {
      marginStart: 8,
    },
```

hi @jpuri, 

Where did you get the design for the UI? 
We could remove our styles in favor of inherit TagBase styles. We could also remove the Text wrapper in TagBase. We should also update marginLeft to marginStart to support RTL languages

```
            <TagBase style={styles.accountLabelWrapper} severity={TagSeverity.Neutral} shape={TagShape.Rectangle}>
              {accountLabel}
            </TagBase>
```

current:
<img width="320" src="https://github.com/user-attachments/assets/04888e9e-002d-4caf-b134-ff17189594f4">

updated w/ shape rectangle or pill (default):
<img width="320" src="https://github.com/user-attachments/assets/f28c2b70-ad26-4c3a-8d75-ec74986da3b4">
<img width="320" src="https://github.com/user-attachments/assets/f5761041-989d-4359-b454-e0d624b61329">

Reply:
PR is updated